src/ai/backend/agent/agent.py

```diff
@@ -696,6 +702,46 @@ async def produce_error_event(
         pretty_tb = "".join(traceback.format_tb(tb)).strip()
         await self.produce_event(AgentErrorEvent(pretty_message, pretty_tb))
 
+    async def _report_all_kernel_commit_status_map(self) -> None:
+        """
+        Commit statuses are managed by `lock` file.
+        +- base_commit_path
+        |_ subdir1 (usually user's email)
+            |_ commit_file1 (named by timestamp)
+            |_ commit_file2
+            |_ lock
+                |_ kernel_id1 (means the user is currently committing the kernel)
+                |_ kernel_id2
+        |_ subdir2
+        ...
+
+        `status_map` is like below.
+        {
+            str(subdir1): {
+                str(kernel_id1): CommitStatus.ONGOING.value,
+                str(kernel_id2): CommitStatus.ONGOING.value,
+            },
+            str(subdir2): ...,
+        }
+        """
+        base_commit_path: Path = self.local_config["agent"]["image-commit-path"]
+        status_map = {}
+        for subdir in base_commit_path.iterdir():
+            ongoing_list = [commit_path.name for commit_path in subdir.glob("./**/lock/*")]
+            status_map[str(subdir)] = {kern: CommitStatus.ONGOING.value for kern in ongoing_list}
```

Comment:
I don't see any counterpart logic of this line on this PR. We need to set back this value to `READY` when container commit is completed, otherwise the API will wait infinitely even if actual commit process is done.

Reply:
I updated the counterpart, by checking remaining kernel status map and update them to `READY` if they do not have lock file